### DISCUSSION TITLE
mixed (zero and first order) absorption model

### DISCUSSION THREAD

**Discussion by harishkaushikbugworks** [nlmixrdevelopment/nlmixr#513] [General]
Hello Matt,

Iam trying to build mixed (zero and first order) absorption model of donepezil from transdermal patch application in nlmixr
I need your help in writing code for modelling these kind of profiles

Iam
[Choi2015_Article_Poppk of donepezil patch.pdf](https://github.com/nlmixrdevelopment/nlmixr/files/6348942/Choi2015_Article_Poppk.of.donepezil.patch.pdf)
 attaching the paper, iam referring to

Regards
HKK

**Comment by mattfidler** [comment]
Typically this is performed by splitting the dose in the dataset into two groups:

1. Group one is to the central compartment;  In nlmixr this can be the name of the compartment like `central` or the compartment number like `1`.  The dose is the total amount of the dose the system. 
2. Group two is to the depot compartment; In nlmixr this can be the name of the compartment like `depot` or the compartment number like `2`.

In the model you would specify the depot compartment like:

```R
model({
d/dt(depot) <- -ka*depot
d/dt(central) <- ka*depot - kel*central
f(central) <- fc
f(depot) <- 1 - fc
})
```

I haven't read the paper, so you should adjust the model to match the paper (ie number of compartments, maybe there is a PD component etc)
